jdaviz/app.py

```diff
@@ -815,26 +773,29 @@ def _get_all_subregions(mask, spec_axis_data):
                     # TODO: this needs to be much simpler; i.e. data units in
                     #  the glue component objects
                     # Cases where there is a single subset
-                    if '_orig_spec' in value.data.meta:  # Hack for Cubeviz WCS propagation
-                        data_wcs = value.data.meta['_orig_spec']
-                    else:
-                        data_wcs = value.data.coords
-
-                    subregions_in_subset = _get_all_subregions(
-                            np.where(value.to_mask() == True)[0], # noqa
-                            data_wcs.spectral_axis)
 
-                    regions[key] = subregions_in_subset
+                    # if '_orig_spec' in value.data.meta:  # Hack for Cubeviz WCS propagation
```

Comment:
Is this no longer needed? It was added in https://github.com/spacetelescope/jdaviz/pull/1480 because not all Cubeviz products have WCS attached.